docs/linux/dm.md

```diff
@@ -35,7 +35,7 @@ creation of a *dm-linear* target device.
 
 ### Example: Creating a Small Host Managed Disk
 
-This example illustrates how to create a small host managed disk using zone
+This example illustrates how to create a small host-managed disk that uses zone
 ranges from a large high capacity host managed disk. The zone information of
```

Comment:
If you add the hyphen here, let's also add one to the "host managed" on the next line :)

Reply:
You are correct, sir.